src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/Views/DynamicTableViewHeadlineWithImageCell.swift

```diff
@@ -30,6 +30,7 @@ class DynamicTableViewHeadlineWithImageCell: UITableViewCell {
 		headlineLabel.text = headline
 		backgroundImageView.image = image
 		topInsetConstraint.constant = topInset
+		imageHeightConstraint.constant = imageHeight
```

Comment:
I think we need to update it in `traitCollectionDidChange()` as well for dynamic type changes

Reply:
Or not, the image height would not impacted by dynamic type 😅